### PR TITLE
UI menubar: re-arrange existing menu items

### PR DESCRIPTION
**Pull request by jp8**
**Short description of changes**

Re-arrange existing menu items to adhere to industry standard UI guidelines.

**Context: Follows from discussion #1915 

**Does this change need documentation? What needs to be documented and how?**

If this gets merged, I will open a pull request to the documentation section.

**Status of this Pull Request**

Working implementation

**What is missing until this pull request can be merged?**

Please test and provide comments.

## Checklist

- [x] I've verified that this Pull Request follows the [general code principles](https://github.com/jamulussoftware/jamulus/blob/master/CONTRIBUTING.md#jamulus-projectsource-code-general-principles)
- [x] I tested my code and it does what I want
- [x] My code follows the [style guide](https://github.com/jamulussoftware/jamulus/blob/master/CONTRIBUTING.md#source-code-consistency) 
- [x] I waited some time after this Pull Request was opened and all GitHub checks completed without errors. 
- [x] I've filled all the content above

### CODE REVIEW

**Comment by jp8** [comment]
Screenshot_2021-07-24_12-06-44
Screenshot_2021-07-24_12-06-51
Screenshot_2021-07-24_12-06-56
Screenshot_2021-07-24_12-07-01
Screenshot_2021-07-24_12-07-04

**Comment by gilgongo** [comment]
> If this gets merged, I will open a pull request to the documentation section.

I'm assuming it will just need perhaps a minor adjustment to this section, is that right?

https://github.com/jamulussoftware/jamuluswebsite/blob/next-release/wiki/en/Software-Manual.md#menu-commands

**Comment by jp8** [comment]
> minor adjustment to this section, is that right?

Yes but I was also expecting I would need to update all screenshots.  However all screenshots in the documentation seem to be broken with a {site.url} in the path

https://github.com/jamulussoftware/jamuluswebsite/blob/next-release/wiki/en/%7B%7Bsite.url%7D%7D/assets/img/en-screenshots/main-screen-medium.png

**Review by ann0see — APPROVED** [review]
Compiled. I think it makes sense to do it like that. Some guides need updating especially if we ask people to set their profile via View>My Profile ;-).

**Comment by ann0see** [comment]
> However all screenshots in the documentation seem to be broken with a {site.url} in the path

No, I think that’s ok. You‘ll need to build the site locally via Jekyll to check everything. The .md files are just the source.

**Comment by gilgongo** [comment]
I sincerely hope we won't be adding any screenshots for something that we can just refer to in `menu > command` format.

**Comment by jp8** [comment]
> adding any screenshots

No I just intended to replace any that show the old UI.  I checked the [manual](https://jamulus.io/wiki/Software-Manual).  There is just one word to change, and potentially one screenshot.

File: src/clientdlg.cpp [hunk 1]
```diff
@@ -274,6 +274,10 @@ CClientDlg::CClientDlg ( CClient*         pNCliP,
     // File menu  --------------------------------------------------------------
     QMenu* pFileMenu = new QMenu ( tr ( "&File" ), this );
 
+    pFileMenu->addAction ( tr ( "&Connection Setup..." ), this, SLOT ( OnOpenConnectionSetupDialog() ), QKeySequence ( Qt::CTRL + Qt::Key_C ) );
```
Comment (pljones) [inline src/clientdlg.cpp:277]: Ctrl-C means Cancel.  It means Cancel on Windows, Linux and MacOS CLI environments.  I'm hesitant about using it for anything else.  (It's bad having to remember Ctrl-Z - background - vs Ctrl-Z - undo...)

Personally, I'd use Ctrl-O (like "open" a new connection).  But I guess it was already Ctrl-C somewhere else...

**Review by pljones — APPROVED** [review]
Apart from my on-going dislike of Ctrl-C/&C for opening a new connection, it's okay.

File: src/clientdlg.cpp (hunk 1 above)
Comment (jp8) [reply src/clientdlg.cpp:277]: Yes it was Ctrl-C before - I left all the keyboard shortcuts exactly as they were.

I am planning to do a sequence of UI -related pull requests.  For the next pull request there are a few menu items I'd like to add, and some other small changes.  For me, that would be the ideal opportunity to make sure everyone is happy with the keyboard shortcuts and the 'menu accelerators'.

I believe the menu accelerator keys change according to the user's language, but the keyboard shortcuts always remain the same - is that right ?

**Comment by jp8** [comment]
For the documentation, shall I:
1. make the screenshot from linux
2. install a windows vm and make the screenshot in windows
3. just leave the existing screenshot in the docs

image

**Comment by ann0see** [comment]
We‘re not consistent. I think

> install a windows vm and make the screenshot in windows

is not needed?

**Pull request merged**